src/Domain/ContentRenderer/CodeRenderer.php

```diff
@@ -11,14 +11,15 @@ class CodeRenderer implements ContentRenderer {
 
 	public function __construct(
 		private string $language,
-		private bool $showLineNumbers
+		private bool $showLineNumbers,
+		private bool $showEditButton = false
```

Comment:
Default config should not be defined in the renderer. This is done in `EmbedUseCase::createRendererConfig()`.